cypress/support/fragments/inventory/instanceRecordView.js

```diff
@@ -482,6 +482,20 @@ export default {
     );
   },
 
+  verifyContributorNameWithoutMarcAppIcon: (indexRow, indexColumn, value) => {
+    cy.expect(
+      Accordion('Contributor')
+        .find(MultiColumnListRow({ index: indexRow }))
+        .find(
+          MultiColumnListCell({
+            columnIndex: indexColumn,
+            content: 'Linked to MARC authority' + value,
+          }),
+        )
+        .absent(),
```

Comment:
This method will be false positive when there is no such value. Need to find MultiColumnList with content value and verify  has({ content: not(including('Linked to MARC authority')) })